ocp_resources/cluster_service_version.py

```diff
@@ -6,3 +8,14 @@ class ClusterServiceVersion(NamespacedResource):
 
     class Status(NamespacedResource.Status):
         INSTALLING = "Installing"
+
+    def get_dicts_from_examples(self):
```

Comment:
the name is confusing as the function returns a list of resource dicts; how about something like `get_resource_examples` ?